docs/source/json-api/index.rst

```diff
@@ -266,12 +266,16 @@ The JSON API essentially performs two separate tasks:
 
 .. note:: By default, the Daml Sandbox does not does not require access tokens. However, you still need to provide a party-specific access token when submitting commands or queries as a party. The token will not be validated in this case but it will be decoded to extract information like the party submitting the command.
 
-Party-specific Access Tokens
-----------------------------
+Party-specific Requests
+-----------------------
 
-Party-specific requests, i.e., command submissions and queries, require a JWT with some additional restrictions compared to general :ref:`Daml ledger access token formats <access-token-formats>`. For command submissions, ``actAs`` must contain at least one party and ``readAs`` can contain 0 or more parties. Queries require at least one party in either ``actAs`` or ``readAs``. The application id and ledger id are mandatory. HTTP requests pass the token in a header, while WebSocket requests pass the token in a subprotocol.
+Party-specific requests, i.e., command submissions and queries, require a JWT that demonstrates the bearer's authorization to issue the request.
```

Comment:
This paragraph is not very informative to me, what is a JWT that demonstrates the bearer's authorization?

And should we still include it like this if the user management shall be the recommended way of dealing with tokens? I.e. shouldn't the user management way be mentioned first and then mentioned as legacy approach the party specific tokens?

PS: Documentation is always very fuzzy thing to solve, these are just my thoughts on this.

Reply:
Is your ask to remove this paragraph altogether or something else? I cannot tell.